interpreter/text/parser.mly

```diff
@@ -39,15 +39,14 @@ let ati i =
 let literal f s =
   try f s with Failure _ -> error s.at "constant out of range"
 
-let simd_literal shape ss =
+let simd_literal shape ss at =
   try
-      let v = V128.of_strings shape (List.map (fun s -> s.it) ss) in
-      (* This List.hd call is okay an exception would have been raise if length is 0. *)
-      (v128_const (v @@ (List.hd ss).at), Values.V128 v)
+    let v = V128.of_strings shape (List.map (fun s -> s.it) ss) in
+    (v128_const (v @@ at), Values.V128 v)
   with
     (* TODO better location for error messages. *)
-    | Failure _ -> error (ati 1) "constant out of range"
-    | Invalid_argument _ -> error (ati 1) "unexpected token"
+    | Failure _ -> error at "constant out of range"
+    | Invalid_argument _ -> error at "unexpected token"
```

Comment:
Nit: "wrong number of lane literals"

Reply:
Fixed, and changed test assertion messages.